Bottom: b58d60e6251f4a2c2d20ef48380f8cbfbd7892a0
Top:    4999d6b3af3cbc347dd8a32a45b8dae1ac82f4e1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-26 16:13:04 +0000

WAS: Use motion hints when processing pointer motion over the drawing area

Not sure what the remaining changes here are for!


---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 7e3c698..bd95bc5 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -750,7 +750,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_LINES);
 
-#if 1
+#if 0
   if (x_prev >= 0)
     draw_crosshair (x_prev, y_prev);
 #endif
@@ -763,7 +763,7 @@ ghid_show_crosshair (gboolean show)
 
   glBegin (GL_QUADS);
 
-#if 1
+#if 0
   if (x_prev >= 0 && draw_markers_prev)
     {
       glVertex2i (0,                  y_prev - VCD);
